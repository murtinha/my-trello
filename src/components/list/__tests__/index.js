import React from 'react';
import TestRenderer from 'react-test-renderer';
import List from '../';


describe('<List />', () => {
  describe('render', () => {
    it('should render properly', () => {
      const comp = TestRenderer
        .create(
          <List title="titulo">
            <div>CHILDREN</div>
          </List>
        )
        .toJSON();
      expect(comp).toMatchSnapshot();
    });
  });
});
