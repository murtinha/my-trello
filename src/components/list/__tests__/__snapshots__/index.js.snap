// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> render should render properly 1`] = `
<div
  className="List"
>
  <div
    className="List-header"
  >
    <div
      className="List-header-titleWrapper"
    >
      <div
        className="List-header-title"
      >
        titulo
      </div>
      <span
        className="List-header-menu fas fa-ellipsis-h"
      />
    </div>
    <div
      className="List-header-amount"
    >
      1 card
    </div>
  </div>
  <div
    className="List-body"
  >
    <div>
      CHILDREN
    </div>
  </div>
</div>
`;
